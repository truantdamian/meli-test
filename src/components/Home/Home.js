import React from 'react'
import './home.scss';

const Home = props => {
    return (
        <div className="section-home">
            <h1>Meli Challenge</h1>
            <h4>Dami√°n Truant</h4>
        </div>
    )
}

export default Home
