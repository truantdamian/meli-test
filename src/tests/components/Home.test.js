import React from 'react';
import { shallow } from 'enzyme';
import Home from './../../components/Home/Home'

describe('tests in Home', () => {
    test('should ', () => {
        const wrapper = shallow(<Home/>);
        expect(wrapper).toMatchSnapshot();
    })
    
});
