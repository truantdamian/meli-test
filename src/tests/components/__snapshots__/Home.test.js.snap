// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in Home should  1`] = `
<div
  className="section-home"
>
  <h1>
    Meli Challenge
  </h1>
  <h4>
    Dami√°n Truant
  </h4>
</div>
`;
